wallet-new/src/Cardano/Wallet/API/V1/Accounts.hs

```diff
@@ -19,7 +19,7 @@ type API
     :<|> "accounts" :> WalletRequestParams
                     :> Summary "Retrieves the full list of Accounts."
                     :> Get '[JSON] (OneOf [Account] (ExtendedResponse [Account]))
-    :<|> "accounts" :> Header  "Daedalus-Passphrase" Text
+    :<|> "accounts" :> Header  "Daedalus-Passphrase" PassPhrase
```

Comment:
Let's use `SpendingPassword`, and rename `Daedalus-Passphrase` into `Daedalus-Spending-Password` or similar.